Doc/faq/programming.rst

```diff
@@ -1036,7 +1036,7 @@ A raw string ending with an odd number of backslashes will escape the string's q
    >>> r'C:\this\will\not\work\'
      File "<stdin>", line 1
        r'C:\this\will\not\work\'
-            ^
+       ^
    SyntaxError: unterminated string literal (detected at line 1)
```

Comment:
On main, this would also result in showing the suggestion:

> SyntaxError: unterminated string literal (detected at line 1); perhaps you escaped the end quote?